testnet/stacks-node/src/config.rs

```diff
@@ -878,6 +970,8 @@ pub struct BurnchainConfigFile {
     pub chain: Option<String>,
     pub burn_fee_cap: Option<u64>,
     pub mode: Option<String>,
+    pub chain_id: Option<String>,
+    pub peer_version: Option<String>,
```

Comment:
This probably shouldn't be configurable.

Reply:
Address with https://github.com/blockstack/stacks-blockchain/pull/2173/commits/a7c78d4e28d380466449a8734056b0d06844f733